config/jobs/cert-manager/cert-manager/release-previous/cert-manager-release-previous-periodics.yaml

```diff
@@ -174,28 +173,40 @@ periodics:
     preset-bazel-remote-cache-enabled: "true"
     preset-bazel-scratch-dir: "true"
     preset-cloudflare-credentials: "true"
-    preset-enable-all-feature-gates: "true"
+    preset-enable-all-feature-gates-disable-ssa: "true"
```

Comment:
It looks like for some reason the SSA tests were *enabled* against 1.8 even on older Kubernetes clusters... not sure why, but this could also be related to the issues we saw

Reply:
Yeah some kind of configuration generator would be great - Kubernetes actually already does this today too: https://github.com/kubernetes/test-infra/blob/master/config/jobs/kubernetes/generated/generated.yaml

Which is generated using: https://github.com/kubernetes/test-infra/tree/master/releng